### ISSUE TITLE
Update ~ 2017-09-29 ~ gbXML Viewer 'Michal'.2

### ISSUE THREAD

**Issue by theo-armour** [ladybug-tools/spider#22]
@ladybug-tools/spiders 

![image](https://user-images.githubusercontent.com/547626/31006534-dc9455ea-a4b1-11e7-8041-2f94f77a7f17.png)

![image](https://user-images.githubusercontent.com/547626/31006555-f09c487c-a4b1-11e7-9826-fa6a74c39fec.png)

## Full Screen: [gbXML Viewer 'Michal'.2 ]( http://ladybug-tools.github.io/spider/read-gbxml/gbxml-viewer/index.html )

**Note: you may need to click the title in the left menu to get the script to to run the first time**

## Change Log
* gbXML Viewer 'Michal'.2
* Huge update! ;-)
* Handles all sorts of polyloop types - with good speed
* And I learned all about conjugating quaternions in the process ;-)

**Comment by mdengusiak** [comment]
I am trying to understand different version or viewer: Michal, Mostapha..tec. Howe does it work. I am not sure I do understand this structure. Are you developing each one as separate ?

**Comment by theo-armour** [comment]
@mdengusiak 

> I am trying to understand different version

I got tired of just giving each release a new number. It just seemed boring. So I thought: why not give each release a code name. And then I thought about code names. So I thought why not have code names that say thank you to contributors.

Yes, it is too confusing. Even for me. In the folder where the releases are kept, I have had to add a date to each folder

https://github.com/ladybug-tools/spider/tree/master/read-gbxml/gbxml-viewer

Once we run out of people or something, we will revert to numbers. OK? ;-)
